app/models/link_checker_api_report/link.rb

```diff
@@ -0,0 +1,16 @@
+class LinkCheckerApiReport::Link < ActiveRecord::Base
+  serialize :check_errors, Hash
+  serialize :check_warnings, Hash
+
+  belongs_to :report, class_name: LinkCheckerApiReport
+
+  def self.attributes_from_link_report(payload)
```

Comment:
Can we use the `LinkReport` model from GDS API Adapters here? Or in similar places?